geoportailv3/static/js/catalog/catalogdirective.js

```diff
@@ -87,6 +87,9 @@ app.CatalogController.prototype.getInfo = function(node, layer) {
  */
 app.CatalogController.prototype.getLayer = function(node) {
   var layer = this.getLayerFunc_(node);
+  if (layer) {
```

Comment:
I'd suggest always use `goog.isNull` or `goog.isDef` to make the code more explicit. In this case I guess what you want is:
 
```
  if (!goog.isNull(layer)) {
    // …
  }
```